### PR TITLE
deprecate unused vela commands

### LINKED ISSUE #644: [Feature] deprecate 'vela svc deploy' and delete the related code
`vela up` should be the only way for deploy apps.

### PR DESCRIPTION
**Pull request by wonderflow**
1. remove vela comp deploy fix #644
2. remove `vela <trait> xxx` 

related issue : #877

### CODE REVIEW

**Comment by ryanzhang-oss** [comment]
Is there anything replace "vela <trait> xx"?

File: Makefile [hunk 1]
```diff
@@ -105,7 +105,7 @@ vet:
 lint: golangci
 	$(GOLANGCILINT) run  ./...
 
-reviewable: manifests fmt vet lint
+reviewable: manifests fmt vet lint doc-gen
```
Comment (zzxwill) [inline Makefile:108]: 改动可以避免我们忘了 doc-gen。+1

但是 1）随着内置 capability 移动出来，需要 doc 跟越来越少 2）如果用户扩展了 capability，会生成 doc，这个 doc 又不能 commit，需要额外手工处理。-1

综合看来，还是不加：）

Comment (wonderflow) [reply Makefile:108]: make sense, will remove it

**Comment by wonderflow** [comment]
> Is there anything replace "vela xx"?

No, we have agreed that use appfile as the  only command line entrypoint

**Review by zzxwill — APPROVED** [review]
LGTM

**Pull request merged**